src/encord_active/lib/db/merged_metrics.py

```diff
@@ -56,7 +56,7 @@ def build_merged_metrics(metrics_path: Path) -> pd.DataFrame:
             main_df_objects.rename(columns={"score": f"{meta['title']}"}, inplace=True)
 
     main_df = pd.concat([main_df_images, main_df_objects, main_df_image_quality])
-    main_df["tags"] = None
+    main_df["tags"] = [[]] * len(main_df)
```

Comment:
This is dangerous! Doing this would use the reference to the same list for all data items. 
```suggestion
    main_df["tags"] = [[] for _ in range(len(main_df))]
```